### DISCUSSION TITLE
Do you have any plans to add file transfer service?

### DISCUSSION THREAD

**Discussion by canibal1** [vavadiyahiren/xmpp_plugin#33] [Ideas]
It is important to be able to send files (video, photo, etc.) in chat. Will you work on this?

**Comment by vavadiyahiren** [comment]
Yes i will be working on this for next sprints.

**Reply by canibal1** [reply]
Thank you. If you want, I can also contribute to the package. However, I have problems reading and developing documentation about xmpp.
